components/server/src/ome/api/local/LocalCompress.java

```diff
@@ -39,15 +39,14 @@ void compressToStream(BufferedImage image, OutputStream outputStream)
 	 * 
 	 * @param percentage A percentage compression level from 1.00 (100%) to 
 	 * 0.01 (1%).
-	 * @throws ValidationException if the <code>percentage</code> is out of
-	 * range.
+	 * @throws ome.conditions.ValidationException if the {@code percentage} is out of
```

Comment:
The `<i>stream<i>` in the previous method lines above alters the rendering of this [page](https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/javadoc/ome/api/local/LocalCompress.html). Not the fault of this PR though - can be fixed in a follow-up PR.